src/marqo/s2_inference/s2_inference.py

```diff
@@ -160,6 +167,52 @@ def get_marqo_inference_cache() -> MarqoInferenceCache:
     return _marqo_inference_cache
 
 
+def is_preprocess_image_model(model_name: str, model_properties: dict = None) -> bool:
+    """Check if the model should be preloaded with an image preprocessor to preprocess image tensor_search module"""
+
+    validated_model_properties = validate_model_properties(model_name, model_properties)
```

Comment:
The Model class already performs this validation

Reply:
done